.github/workflows/publish_world_flags.yaml

```
@@ -0,0 +1,16 @@
+    branches: [main]
+    tags:
+    - 'world_flags-v[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
+  publish-world-flags:
+    uses: ./.github/workflows/publish_to_pub.yaml
+    permissions:
+      id-token: write
+    with:
+      package_name: world_flags
+      tag: ${{ github.ref }}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)